packaging/linux/linux_agent_builder.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -eux
```

Comment:
We don't technically need it yet, but it might be worth adding the pipefail option to this so we have just one standard invocation that we can copy-paste everywhere and get well behaved scripts.